### PR TITLE
Configure helm-broker repository baseURL to github pages, remove embedded reposerver

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- Set Helm-broker repository url to github pages: kyma-incubator/kyma-bundles
- Remove helm-broker-reposerver

**Related issue(s)**
Related: #592

### CODE REVIEW

File: resources/README.md [hunk 1]
```diff
@@ -17,5 +17,4 @@ Resources currently include, but are not limited to, the following:
 - Elements which are essential for the installation of `core` components in Kyma, such as certificates, users, and permissions
 - The `core` components required to run the Kyma
 - Examples of the use of specific components
-- The repository of bundles that are exposed in the Service Catalog as Service Classes through the Helm Broker
```
Comment (mszostok) [inline resources/README.md]: please also adjust the documentation in this file:
- docs/service-brokers/docs/004-overview-helm-broker.md
- docs/service-brokers/docs/011-configuration-helm-broker.md
- docs/service-brokers/docs/021-architecture-helm-broker.md

File: docs/service-brokers/docs/004-overview-helm-broker.md [hunk 2]
```diff
@@ -3,6 +3,6 @@ title: Helm Broker
...
 ---
 
-The Helm Broker is an implementation of a service broker which runs in the Kyma cluster and deploys Kubernetes native resources using [Helm](https://github.com/kubernetes/helm) and Kyma bundles. A bundle is an abstraction layer over a Helm chart which allows you to represent it as a ClusterServiceClass in the Service Catalog. For example, a bundle can provide plan definitions or binding details. The Helm Broker fetches bundle definitions from an HTTP server. By default, the Helm Broker contains an embedded HTTP server which serves bundles from the Kyma bundles directory.
+The Helm Broker is an implementation of a service broker which runs in the Kyma cluster and deploys Kubernetes native resources using [Helm](https://github.com/kubernetes/helm) and Kyma bundles. A bundle is an abstraction layer over a Helm chart which allows you to represent it as a ClusterServiceClass in the Service Catalog. For example, a bundle can provide plan definitions or binding details. The Helm Broker fetches bundle definitions from an HTTP server. By default, the Helm Broker fetches bundles from the [stable kyma-bundles repository](https://kyma-incubator.github.io/kyma-bundles/stable/).
```
Comment (klaudiagrz) [inline docs/service-brokers/docs/004-overview-helm-broker.md]: 1. `kyma-bundles`
2. Fix a broken link

File: docs/service-brokers/docs/011-configuration-helm-broker.md [hunk 3]
```diff
@@ -42,17 +33,12 @@ Follow these steps to change the configuration and make the Helm Broker fetch bu
...
-  # To provision, specify this value to true
-  provision: true
-
   config:
     repository:
       baseURL: "http://custom.bundles-repository"
   ```
 
-3. Install Kyma on Minikube. See the **Local Kyma installation** document to learn how.
+2. Install Kyma on Minikube. See the **Local Kyma installation** document to learn how.
```
Comment (klaudiagrz) [inline docs/service-brokers/docs/011-configuration-helm-broker.md]: `3.` -> please fix the numeration

File: resources/README.md (hunk 1 above)
Comment (piotrmiskiewicz) [reply resources/README.md]: fixed

File: resources/core/charts/helm-broker/values.yaml [hunk 4]
```diff
@@ -8,19 +8,10 @@ service:
...
-  # Defines service for embedded repository
-  service:
-    type: NodePort
-    externalPort: 80
-    internalPort: 8080
 
 config:
   repository:
-    baseURL: "http://core-helm-broker-reposerver"
+    baseURL: "https://kyma-project.github.io/kyma-bundles/stable/"
```
Comment (PK85) [inline resources/core/charts/helm-broker/values.yaml]: to: https://kyma-project.github.io/bundles/stable/

Comment (piotrmiskiewicz) [reply resources/core/charts/helm-broker/values.yaml]: changed

**Pull request merged**